UM-USElab/gradecraft-development
# Added adjustment points and final score to challenge grades

### Status
**READY**

### Description
This PR adds the adjustment points functionality to `ChallengeGrades` 

### Migrations
YES

### Steps to Test or Reproduce
1. Pull down the branch and run migrations
2. As an instructor, edit a Challenge Grade to add adjustment points and feedback
3. Log in as a student in the team whose Challenge Grade you edited
4. Navigate to the teams page on the student side
5. Observe that your challenge grade's final score takes into account the adjustment points
6. Click the link to see the detailed feedback and see the explicit adjustment points and feedback

### Impacted Areas in Application
List general components of the application that this PR will affect:
* Challenge Grades
